pkg/reconciler/taskrun/taskrun.go

```diff
@@ -289,6 +291,7 @@ func (c *Reconciler) prepare(ctx context.Context, tr *v1beta1.TaskRun) (*v1beta1
 		return nil, nil, err
 	}
 
+	c.updateTaskRunWithDefaultWorkspaces(ctx, tr, taskSpec)
```

Comment:
This function could return an error so I wonder if we should log it and mark the taskrun as failed if that happens. What do you think?

Reply:
Agreed, will add that, thanks!

Reply:
Done ✅